.github/actions/run-python-test-set/action.yml

```diff
@@ -27,6 +27,26 @@ inputs:
     description: 'Whether to upload the performance report'
     required: false
     default: 'false'
+  run_with_real_s3:
```

Comment:
Yep, I think everything other than moto or local fs can be called real :) I'm open to better naming suggestion though. Maybe real_remote_storage instead? It is a bit more vague but because of that can cover wider spectrum of possible implementations

Reply:
I would forget that there's anything "real" and just use "local fs" and "s3" concepts everywhere but one single place: somewhere deep in Python test env builder, where we could specify a storage type as Moto and that would run an extra process.

Otherwise, moto or not, IMO it's better to work with either as regular S3.

Reply:
It would be good to avoid the distinction between kinds of s3, but for moto vs something else we need that to know whether to start moto server or provide credentials for example. 

To me this largely depends on how we will add new implementations to tests. If we will add them via pytest's parametrization e g something like gcp_s3, minio etc then we can mention them directly here. So REAL_S3 -> AWS_S3 Other option is to use generic s3 and one flavour but instead run the same test pack with different credentials. WDYT?

Reply:
I like the generic option since that's what I've argumented for 🙂

Reply:
I'm partially on board with that. Though I think it will complicate testing workflow. If you want to run a test locally for all kinds of remote storages how that will look like? If pytest knows only about one s3 you'll need to orchestrate that manually via several pytest invocations. Also I think this is rather incompatible with the way how we run moto. Because it means that we need to run it externally and pass credentials to pytest via env vars. Am I missing something?

Reply:
Might be too much of a hassle indeed.
Never mind me then and thanks for considering.